masonry_testing/src/screenshots.rs

```diff
@@ -24,12 +24,8 @@ pub(crate) fn get_image_diff(
     new_image: &RgbImage,
     max_screenshot_tolerance: u32,
 ) -> Option<RgbImage> {
-    // TODO - Handle this case more gracefully.
-    assert_eq!(
-        (ref_image.width(), ref_image.height()),
-        (new_image.width(), new_image.height()),
-        "New image (right) has different size from old image (left)."
-    );
+    let ref_size = (ref_image.width(), ref_image.height());
+    let new_size = (new_image.width(), new_image.height());
```

Comment:
This was an easy lift, as the diff image computation already supported differently-sized images.